README.md

```diff
@@ -210,40 +205,42 @@ Follow the howto article [HOW-to guides: Manage Models in the repository](https:
 
 Based on the existing samples, create a new device to implement the [Themorstat Model](./models/Thermostat.json).
 
-The device should imlement the next interfaces:
+The device should implement the next interfaces:
+
+- Handle the `reboot` command and send the `workingset` as telemetry in the `Diagnostics` interface
```

Comment:
Are we doing both thermostate and diagnistics in bug-bash?   Could we maybe have them bring up one interface at a time complete, so 1st diagnostics and only once it's good the thermostat (or vice versa on ordering is fine).  This way if they only have time for one, they at least get some e2e flow going and feedback on one path.

Reply:
adding a note, although I think will be valuable to see how folks try to break down the task list.